src/provider/element-templates/ElementTemplates.js

```diff
@@ -196,6 +203,50 @@ export default class ElementTemplates {
 
     return context.element;
   }
+
+  /**
+   * Remove template from a given element.
+   *
+   * @param {djs.model.Base} element
+   */
+  removeTemplate(element) {
+    removeTemplateHelper(element, this._injector);
+  }
+
+  /**
+   * Unlink template from a given element.
+   *
+   * @param {djs.model.Base} element
+   */
+  unlinkTemplate(element) {
+    this._eventBus.fire('elementTemplates.unlink', { element });
+
+    this._modeling.updateProperties(element, {
+      'camunda:modelerTemplate': null,
+      'camunda:modelerTemplateVersion': null
+    });
+  }
+
+  /**
+   * Update template to latest version.
+   *
+   * @param {djs.model.Base} element
+   * @param {ElementTemplate} newTemplate
+   * @returns {djs.model.Base} the updated element
+   */
+  updateTemplate(element, newTemplate) {
+    this._eventBus.fire('elementTemplates.update', { element, newTemplate });
+
+    return this.applyTemplate(element, newTemplate);
```

Comment:
Same like above; if it is all just "applying a new template", then we could enhance whatever is emitted in `applyTemplate`, to distinguish apply and update.